src/e2e/specs/breachResolution.spec.ts

```diff
@@ -168,21 +96,19 @@ test.describe("Breaches Dashboard - Headers", () => {
     const links = dataBreachPage.dashboardLinks();
 
     // verify the navigation within monitor
-    await expect(async () => {
-      // settings button redirects the user to "Settings" tab
-      expect(await dataBreachPage.settingsNavButton.getAttribute("href")).toBe(
-        links.settingsNavButtonLink,
-      );
-
-      // redirects the user to the "Resolve data breaches" tab
-      expect(
-        await dataBreachPage.resolveDataBreachesNavButton.getAttribute("href"),
-      ).toContain(links.resolveDataBreachesNavButtonLink);
-
-      // opens a new tab in which user is redirected to the "Monitor Help" page
-      expect(
-        await dataBreachPage.helpAndSupportNavButton.getAttribute("href"),
-      ).toContain(links.helpAndSupportNavButtonLink);
-    }).toPass();
+    // settings button redirects the user to "Settings" tab
+    expect(await dataBreachPage.settingsNavButton.getAttribute("href")).toBe(
+      links.settingsNavButtonLink,
+    );
```

Comment:
For these attribute tests, there's also a more specific matcher [`toHaveAttribute`](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-attribute) for (presumably) better error messages on tests failures. Also applies to the instances below and in the other spec files:

```suggestion
    expect(dataBreachPage.settingsNavButton).toHaveAttribute(
      "href",
      links.settingsNavButtonLink,
    );
```